packages/material-ui/src/Select/Select.test.js

```diff
@@ -343,7 +343,7 @@ describe('<Select />', () => {
             <MenuItem value={30}>Thirty</MenuItem>
           </Select>,
         );
-        expect(console.warn.callCount).to.equal(1);
+        expect(console.warn.callCount).to.equal(2);
```

Comment:
What's the second message?

Reply:
The same one, strict more `childrenArray` reference change.